aleph0/src/lib/file.ts

```diff
@@ -6,6 +6,11 @@ export function readFile(filePath: string) {
   return fs.readFileSync(filePath, "utf-8");
 }
 
+export async function readMetadata(filePath: string) {
+  const metadata = await import(path.join("..", "..", filePath)); // TODO: fix this
+  return JSON.stringify(metadata);
+}
```

Comment:
You can pass in root dir which might be `process.cwd` by default.

Note, that the root for metadata and the root for the project could be different folders. Similar to how we have now that the folders are in different places.

Reply:
What do you mean exactly?

Reply:
Path is hard coded to ../.. atm. I guess that's fine for now because it's all within our project. If a user decides to create their own snippets the location of these may change. So you'd pass that in as an argument to the cli.

Reply:
You mean pass process.cwd()? Can you write here a suggestion on how you see it?

Reply:
Related to https://github.com/alephmatic/aleph0/issues/51